tests/unit/mixins/sortable-item-test.js

```diff
@@ -103,8 +103,9 @@ test('registers itself with group', function(assert) {
 });
 
 test('deregisters itself when removed', function(assert) {
-  subject.remove();
-
+  run(() => {
+    subject.remove();
+  });
```

Comment:
Was this test failing without the runloop? It doesn’t seem like the patch would require this change.